### PR TITLE
add SearchEmptyScreen that displays when the search for sessions hits no result.

### LINKED ISSUE #597: Add empty  search screen function when no exists results
**Idea Description**
The title says everything 🙏 

**Reference images and links**
https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=0%3A1
image

### PR DESCRIPTION
**Pull request by nyafunta9858**
## Issue
- close #597

## Overview (Required)
- add SearchEmptyScreen that displays when the search for sessions hits no result.
- add the judgment whether showing SearchedItemListField or SearchEmptyScreen by referring to DroidKaigiSchedule.

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
@nyafunta9858  Please run ./gradlew spotlessKotlinApply 🙏

**Comment by takahirom** [comment]
I told the center problem to the designer. Please wait 🙏

**Comment by takahirom** [comment]
I'm waiting for the reply, but I think we can merge this because it is better than the old screen!!

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt [hunk 1]
```diff
@@ -516,3 +557,24 @@ fun SearchScreenPreview() {
         )
     }
 }
+
+@Preview(showSystemUi = false)
+@Composable
+fun SearchEmptyScreenPreview() {
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:563]: 💯

**Comment by takahirom** [comment]
Thanks for your contribution! I'll let you know if I get the information!

**Comment by nyafunta9858** [comment]
Thanks for your kind follow-up! And I'm so glad to contribute to this issue. 🙏

**Pull request merged**